migrations/Version20240204141515.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240204141515 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('CREATE EXTENSION IF NOT EXISTS citext');
+        $this->addSql('ALTER TABLE "user" ALTER COLUMN username TYPE citext');
+    }
+
+    public function down(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE "user" ALTER COLUMN username TYPE VARCHAR(35)');
+        $this->addSql('DROP EXTENSION citext');
```

Comment:
I am not sure if we should drop the extension here. It doesn't hurt to have it and we don't know if we added it in the upper statement, maybe it already existed

Reply:
but it's part of `down` right? You clean it up again since on line 22 you add it on `up`.

Reply:
Well the `up` contains a `IF NOT EXISTS`, so it might have already been present on the machine. I think it could break some pg setups (if this migration is reverted) if `citext` is needed somewhere else (not in mbin), and was therefore already installed...

Reply:
Ah yes. I got it.

Reply:
@thepaperpilot if you remove this line I'll approve, the rest is looking good to me